### DISCUSSION TITLE
Why is the cache folder empty

### DISCUSSION THREAD

**Discussion by MyPrototypeWhat** [vercel/turborepo#6752] [Help]
<img width="1054" alt="image" src="https://github.com/vercel/turbo/assets/43230886/b6da3517-5635-47eb-886b-f0de3d5ed89c">

`./.turbo-cache` This folder is empty

```
ls -trl ./.turbo-cache ===> total 0
```

**Comment by anthonyshew** [comment]
Are the outputs of your build captured by [the `outputs` key in `turbo.json`](https://turbo.build/repo/docs/reference/configuration#outputs)?
